decksite/resources.json

```diff
@@ -17,7 +17,8 @@
     "Deck checker": "http://pennydreadfulmagic.com/deckcheck/",
     "Decklists": "https://pennydreadfulmagic.com/",
     "Legal cards list": "http://pdmtgo.com/legal_cards.txt",
-    "Intro to brewing": "https://www.reddit.com/r/PennyDreadfulMTG/comments/8yy2q3/newb_guide_how_to_start_brewing_in_penny_dreadful/"
+    "Intro to brewing": "https://www.reddit.com/r/PennyDreadfulMTG/comments/8yy2q3/newb_guide_how_to_start_brewing_in_penny_dreadful/",
+    "Frank Karsten manabase": "https://www.channelfireball.com/all-strategy/articles/how-many-colored-mana-sources-do-you-need-to-consistently-cast-your-spells-a-guilds-of-ravnica-update/"
```

Comment:
Include a trailing comma for nicer diffs next time

Reply:
Sadly that's not legal JSON. Blame Douglas Crockford …